packages/editor/src/components/post-actions/actions.js

```diff
@@ -1048,7 +1078,7 @@ export function usePostActions( { postType, onActionPerformed, context } ) {
 			isTemplateOrTemplatePart ? resetTemplateAction : restorePostAction,
 			isTemplateOrTemplatePart || isPattern
 				? deletePostAction
```

Comment:
Don't we need the same for `deletePostAction`?

Reply:
Yes, we need the same for some other actions, my plan was to define the fix here and after we agree on a fix I will replicate it to the other cases where it makes sense.

Reply:
Is permanently delete a separate capability? I'm still seeing this:

<img width="380" alt="image" src="https://github.com/WordPress/gutenberg/assets/4710635/90fda1d1-f781-48d0-b349-58a14952bb68">

Reply:
I tested with the testing instructions, and it seems to be the same capability: I can also not delete it permanently in trunk.